android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -819,35 +819,48 @@ private Integer getPidForProcessName(String processName) {
     	
     	// figure out what index PID has
     	String [] headers = processLinesAr[0].split("[\\s]+");
-    	Integer PidIndex = 1;
+    	Integer PidIndex = -1;
     	for (int x = 0; x < headers.length; x++) {
     		if(headers[x].equals("PID")) {
     			PidIndex = x;
     			break;
     		}
     	}
-		if(Logging.DEBUG) Log.d(Logging.TAG,"getPidForProcessName(): PID at index: " + PidIndex + " for output: " + processLinesAr[0]);
-    	
+
+		if (PidIndex == -1)
```

Comment:
Please review the indentation of all lines between 829 and 864. At least GitHub shows things are really off right now. Thanks.